_posts/2017-06-23-announcing-angular-master-class-in-houston.md

```diff
@@ -29,41 +24,41 @@ related_videos:
 ---
 
 
-While we just [returned from Denmark](/announcements/2017/05/05/announcing-angular-master-class-in-denmark.html) and announced another upcoming [Angular Master Class in Berlin](/announcements/2017/06/23/announcing-angular-master-class-in-berlin.html), we know that people want us to run our training all around the world.
-
 It's been a couple of month since we've last run our training in the US so we are very pleased to announce that our **Angular Master Class is coming to Houston in August!**
 
-We've found the perfect, artsy venue to get into the right piece of mind to learn Angular in and out.
+ We've found the perfect, artsy venue to get into the right piece of mind to learn Angular in and out. The event takes place on the **22nd to 25th August 2017** in the **Bisong Art Gallery**.
 
 <img src="/images/amc_houston_hall.jpg" alt="AMC Houston Event Location">
 
-The event takes place on the **22nd to 25th August 2017** in the **Bisong Art Gallery**.
+This is the ONLY public training thoughtram will provide [in Texas] for at least the next 12 months... so do not miss this opportunity.
 
-This is the first time we'll run a 4-day training. The reason we added another day is because we want to teach our brand new material on  **Redux and ngrx**. Join us for a couple of days of learning and fun including things like
+You or your team can attend our public 3­-day or 4­-day training.
 
-- Components and Modules
-- Fetching data using Http
-- Advanced Dependency Injection
-- Basic Forms
-- Observables deep-dive
-- Architectural patterns
-- ContentChildren and ViewChildren
-- Template-driven Forms
-- Reactive Forms
-- Observables deep-dive
-- Children routes and lazy-loading
-- **Redux and ngrx**
+● Day 1: Jump Start with full, high­level introduction to Angular
+● Day 2: Observables & Component Architecture
+● Day 3: Routing & Forms
+● Day 4: Redux + ngrx ­with immutable, 1­-way dataflows and reactive programming
 
-...and many more!
+> If you would like private, onsite training for a team of 10 or more developers, please contact us at training@thoughtram.io.
```

Comment:
We should add an alias for `training@thoughtram.io` as that is an intuitive email IMO.